include/RandLAPACK/comps/orth.hh

```diff
@@ -8,136 +8,144 @@ namespace RandLAPACK::comps::orth {
 #ifndef ORTH_CLASS
 #define ORTH_CLASS
 
+enum decision_orth_stab {use_CholQRQ, use_HQRQ, use_PLUL, use_GEQR};
+
 template <typename T>
-class Stabilization
-{
-        public:
-                virtual int call(
-                        int64_t m,
-                        int64_t k,
-                        std::vector<T>& Q
-                ) = 0;
+class Stabilization {
```

Comment:
this class needs doxygen style documentation,  as do all classes and functions in the code base. @rileyjmurray  this is a larger issue that has been neglected form day one. the problem is growing and needs to be addressed. I suggest that as we modify or add a class of function we add this documentation

Reply:
Thanks for pointing this out. I agree that it needs to be addressed. Ordinarily, I agree with your suggested approach as well. Still, since this PR is so big, I think it might make more sense to get it in and then have @TeachRaccooon's immediate next priority be adding the documentation in another PR.

Reply:
that sounds like a reasonable approach!